docs/workforce-management-framework/topics/con-raincatcher-core-getting-started.adoc

```diff
@@ -0,0 +1,143 @@
+[[id='raincatcher-core-getting-started-{chapter}']]
+= RainCatcher-core Getting Started
+
+Getting started with the Feedhenry RainCatcher Core Repository
```

Comment:
@jstaffor - do you think our documentation should duplicate README or we should point out to README files?  source of this is here: